src/main/java/de/danielbechler/diff/access/PropertyAwareAccessor.java

```diff
@@ -29,4 +29,8 @@
 	Set<Annotation> getReadMethodAnnotations();
 
 	<T extends Annotation> T getReadMethodAnnotation(Class<T> annotationClass);
+
+   <T extends Annotation> T getAnnotation(Class<T> annotationClass);
+
+   int getFieldModifiers();
```

Comment:
Sorry, I was not sure if this isn't a private thread but if it is public I keep on in English.

I suppose you are talking here about the getFieldModifiers() method? (not sure which of the two). I fear to handle this as a config setting in Introspector is beyond my capability without studying deeply and getting explained the internals. The structure is not very obvious and intuitive for me. You see, I use the object differ presently like this:
```
objectDiffer = ObjectDifferBuilder.startBuilding().inclusion().resolveUsing(new CibetInclusionResolver())
               .and().differs().register(new PrimitiveArrayDifferFactory()).build();
```
with CibetInclusionResolver like this:
```
   @Override
   public Inclusion getInclusion(DiffNode node) {

      if (node.getAnnotation(Version.class) != null) {
         return Inclusion.EXCLUDED;
      }

      if (node.getAnnotation(Transient.class) != null) {
         return Inclusion.EXCLUDED;
      }

      if (Modifier.isStatic(node.getFieldModifiers())) {
         return Inclusion.EXCLUDED;
      }

      if (Modifier.isTransient(node.getFieldModifiers())) {
         return Inclusion.EXCLUDED;
      }

      return Inclusion.DEFAULT;
   }
```
the object differ configuration line is difficult to understand without going with the finger through the source code. I guess you want to extend this line of code with a configuration for the introspector something like this:
```
objectDiffer = ObjectDifferBuilder.startBuilding().inclusion().resolveUsing(new CibetInclusionResolver())
               .and().differs().register(new PrimitiveArrayDifferFactory()).and().introspection().registerExtension(new TransientExtension()).build();
```
where however the second and() is missing presently. But maybe I am totally misrouted. As I said I have difficulties to understand the configuration architecture, it is not intuitive for me.

I will answer the other comments later ...